include/mbgl/util/client_options.hpp

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include <memory>
+#include <string>
+
+namespace mbgl {
+
+/**
+ * @brief Holds values for client options.
+ */
+class ClientOptions final {
+public:
+    /**
+     * @brief Constructs a ClientOptions object with default values.
+     */
+    ClientOptions();
+    ~ClientOptions();
+
+    ClientOptions(ClientOptions&&) noexcept;
```

Comment:
What does the `&&` operator do? Thanks for helping me learn c++...

Reply:
https://stackoverflow.com/questions/5481539/what-does-t-double-ampersand-mean-in-c11

Reply:
Thank you @ntadej